### ISSUE TITLE
Annotated with single type argument is not allowed

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#1349]
The following type annotation should generate an error:

```python
x: Annotated[int]
```

**Comment by erictraut** [comment]
This is fixed in Pyright 1.1.100, which I just published. It will also be included in the next release of Pylance.
